### PR TITLE
docs(angular/table): add example for expandable table

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/components-examples/angular/table/expandable-table/expandable-table-example.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<p>This table uses the when predicate for expanding and collapsing detail columns on row click.</p>
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.html:1]:
```suggestion
<p>This table uses the 'when'-predicate for expanding and collapsing detail columns on row click.</p>
```

File: src/components-examples/angular/table/expandable-table/expandable-table-example.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <th sbb-header-cell *sbbHeaderCellDef>Verbindung Bern - Olten</th>
+      <td sbb-cell *sbbCellDef="let element">{{ element.connection }}</td>
+    </ng-container>
+
+    <ng-container sbbColumnDef="duration">
+      <th sbb-header-cell *sbbHeaderCellDef>Dauer</th>
+      <td sbb-cell *sbbCellDef="let element">{{ element.duration }}</td>
+    </ng-container>
+
+    <!-- Expanded content column - The detail row is made up of this one column -->
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.html:20]:
```suggestion
    <!-- Expanded content column - The detail row is made up of this single column -->
```

File: src/components-examples/angular/table/expandable-table/expandable-table-example.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<p>This table uses the when predicate for expanding and collapsing detail columns on row click.</p>
+
+<sbb-table-wrapper>
+  <table sbb-table [dataSource]="dataSource" [multiTemplateDataRows]="true">
+    <ng-container sbbColumnDef="time">
+      <th sbb-header-cell *sbbHeaderCellDef>Zeit</th>
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.html:6]:
```suggestion
      <th sbb-header-cell *sbbHeaderCellDef>Time</th>
```

File: src/components-examples/angular/table/expandable-table/expandable-table-example.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      <td sbb-cell *sbbCellDef="let element">{{ element.time }}</td>
+    </ng-container>
+
+    <ng-container sbbColumnDef="connection">
+      <th sbb-header-cell *sbbHeaderCellDef>Verbindung Bern - Olten</th>
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.html:11]:
```suggestion
      <th sbb-header-cell *sbbHeaderCellDef>Connection Bern - Olten</th>
```

File: src/components-examples/angular/table/expandable-table/expandable-table-example.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      <td sbb-cell *sbbCellDef="let element">{{ element.connection }}</td>
+    </ng-container>
+
+    <ng-container sbbColumnDef="duration">
+      <th sbb-header-cell *sbbHeaderCellDef>Dauer</th>
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.html:16]:
```suggestion
      <th sbb-header-cell *sbbHeaderCellDef>Duration</th>
```

File: src/components-examples/angular/table/expandable-table/expandable-table-example.ts [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+export class ExpandableTableExample {
+  displayedColumns: string[] = ['connection', 'time', 'duration'];
+  dataSource = TABLE_EXAMPLE_DATA;
+  selectedId: number;
+
+  isExpansionDetailRow = (i: number, row: Object) => row.hasOwnProperty('detailRow');
+  isNotExpansionDetailRow = (i: number, row: Object) => !row.hasOwnProperty('detailRow');
+}
+
+const TABLE_EXAMPLE_DATA = [
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.ts:21]: Can you please translate example data in english? thank you :)

File: src/components-examples/angular/table/expandable-table/expandable-table-example.ts [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    duration: '47 min',
+  },
+  { id: 2, detailRow: true, details: 'Bern, Gleis 10; Olten, Gleis 10' },
+  {
+    id: 3,
+    connection: 'IR 16 Richtung Zürich HB',
+    time: '11:33 - 12:00',
+    duration: '27 min',
+  },
+  { id: 3, detailRow: true, changes: 0, details: 'Bern, Gleis 12; Olten, Gleis 4' },
```
Comment (jeripeierSBB) [inline src/components-examples/angular/table/expandable-table/expandable-table-example.ts:42]: Maybe one item should be not expansible to demonstrate the function above?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
I think it would be better to let the expanded state be toggleable, what do you think?

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**